frontend/compositions/visualizations/map/Map.tsx

```diff
@@ -43,88 +61,73 @@ export default function Map() {
         .scale(1 / maxBound)
         .translate(-(x0 + x1) / 2, -(y0 + y1) / 2)
     },
-    [path]
+    [getHeightAndWidth, path]
   )
 
   const moveMap = useCallback(
     (transform: ZoomTransform, transition = true, pointer?: PointCoord) => {
       const transformStr = transform.toString()
-      if (transformStr.includes("NaN")) return
+      if (transformStr.includes("NaN")) {
+        return
```

Comment:
SRP-wise I'm generally a fan of avoiding null/duplicate return statements in any given function - typically you can just check the opposite case before calling the rest of the code (either by wrapping it in the conditional or more ideally checking the condition before invocation) Nonblocker tho!